lib/import/admin.js

```diff
@@ -150,9 +192,13 @@ function save_features(feature_collection) {
  * @return{Promise} Fulfilled when documents are saved.
  */
 function import_admins(country_code, file, verbose) {
-  return read_jsonfile(file, verbose)
-  .then(function(feature_collection) {
-    return enrich_features(country_code, feature_collection, verbose);
+  var worldpop_file = './data/population/worldpop_' + country_code + '.csv';
```

Comment:
Don't construct this file name. Pass it in, just like the geojson file.

Rename `file` to something like `admin_geojson_file`, and then pass it the worldpop file as `worldpop_csv_file` or something.